Library/Homebrew/cmd/search.rb

```diff
@@ -1,14 +1,17 @@
-#:  * `search`, `-S`:
+#:  * `search`:
 #:    Display all locally available formulae for brewing (including tapped ones).
 #:    No online search is performed if called without arguments.
 #:
-#:  * `search`, `-S` <text>|`/`<text>`/`:
+#:  * `search` (`--desc`) <text>|`/`<text>`/`:
 #:    Perform a substring search of formula names for <text>. If <text> is
 #:    surrounded with slashes, then it is interpreted as a regular expression.
 #:    The search for <text> is extended online to some popular taps.
 #:
 #:  * `search` (`--debian`|`--fedora`|`--fink`|`--macports`|`--opensuse`|`--ubuntu`) <text>:
 #:    Search for <text> in the given package manager's list.
+#:
+#:alias: `-S`
+#:
```

Comment:
We'll need to find another solution for documenting the alias, as this breaks our man page that is auto-generated from this comment. (But I don't have an idea yet what a good solution for this would be.) The least invasive, but probably a bit confusing variant would be to make all of the above items start with:

```
(`search`|`-S`)
```

Instead of:

```
`search`, `-S`
```